pkg/sources/github/github.go

```diff
@@ -16,14 +16,15 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/gobwas/glob"
 	"github.com/google/go-github/v63/github"
-	"github.com/trufflesecurity/trufflehog/v3/pkg/handlers"
 	"golang.org/x/exp/rand"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
 
+	"github.com/trufflesecurity/trufflehog/v3/pkg/handlers"
+
 	"github.com/trufflesecurity/trufflehog/v3/pkg/cache"
```

Comment:
**Extreme nit:** Maybe group this import with the ones below it since they're all importing `trufflesecurity/trufflehog/...`